platform/libs.asm/nbproject/project.properties

```diff
@@ -16,10 +16,11 @@
 # under the License.
 
 javac.compilerargs=-Xlint -Xlint:-serial
-javac.source=1.8
+javac.source=11
+javac.target=11
```

Comment:
this might cause problems since some tests are not ready yet for JDK 11, see #4904

i added the `all-tests` label since asm is used in many places - just to be sure.

Reply:
Will push the change to version 8 and see if the tests go green.

Reply:
ASM declares itself compatible with 1.5: https://asm.ow2.io/faq.html#Q14 so there is no need to change the versions.